problems_1/problems_1.tex

```diff
@@ -0,0 +1,264 @@
+
+\smallskip
+
+\item[8.] Высветліце, ці справядлівыя наступныя лагічныя вынікі: \\
+\ruText{ Выясните, верны ли следующие логические следования: }
+\begin{itemize}
+\item[(а)] $\boldsymbol{\mathrm{A}} \to \boldsymbol{\mathrm{B}}$,\,
+$\boldsymbol{\mathrm{D}} \to \overline{\boldsymbol{\mathrm{C}}}$,\,
+$\boldsymbol{\mathrm{C}} \vee \overline{\boldsymbol{\mathrm{B}}}$\,
+$\vDash \boldsymbol{\mathrm{A}} \to \overline{\boldsymbol{\mathrm{D}}}$;
+
+\item[(б)] $\boldsymbol{\mathrm{A}} \to \boldsymbol{\mathrm{B}}$,\, $((\boldsymbol{\mathrm{A}}
+\vee \boldsymbol{\mathrm{D}}) \cdot \boldsymbol{\mathrm{C}}) \to \boldsymbol{\mathrm{E}}$,\,
+$\boldsymbol{\mathrm{D}} \to \boldsymbol{\mathrm{C}}$\, $\vDash ((\boldsymbol{\mathrm{A}} \vee \boldsymbol{\mathrm{D}}) \cdot \boldsymbol{\mathrm{B}}) \to \overline{\boldsymbol{\mathrm{E}}}$;
+
+\item[(в)] $(\boldsymbol{\mathrm{A}} \vee \boldsymbol{\mathrm{B}}) \to (\boldsymbol{\mathrm{C}} \cdot
+\boldsymbol{\mathrm{D}})$,\, $(\boldsymbol{\mathrm{D}} \vee \boldsymbol{\mathrm{E}}) \to
+\boldsymbol{\mathrm{F}}$\, $\vDash \boldsymbol{\mathrm{A}} \to \boldsymbol{\mathrm{F}}$;
+
+\item[(г)] $(\boldsymbol{\mathrm{A}} \cdot \boldsymbol{\mathrm{B}}) \to \boldsymbol{\mathrm{C}}$,\,
+$(\boldsymbol{\mathrm{C}} \cdot \boldsymbol{\mathrm{D}}) \to \boldsymbol{\mathrm{E}}$,\,
+$\overline{\boldsymbol{\mathrm{F}}} \to (\boldsymbol{\mathrm{D}} \cdot \boldsymbol{\mathrm{E}})$\,
+$\vDash (\boldsymbol{\mathrm{A}} \cdot \boldsymbol{\mathrm{B}}) \to \boldsymbol{\mathrm{F}}$.
+\end{itemize}
+
+\smallskip
+
+\item[9.] Высветліце, ці справядлівыя наступныя сцверджанні: \\
+\ruText{ Выясните, верны ли следующие утверждения: }
+\begin{itemize}
+\item[(а)] калі\, $\boldsymbol{\Gamma} \vDash \boldsymbol{\mathrm{A}}$\, і\, $\boldsymbol{\Gamma}
+\vDash \boldsymbol{\mathrm{B}}$,\, то\, $\boldsymbol{\Gamma} \vDash \boldsymbol{\mathrm{A}} \cdot
+\boldsymbol{\mathrm{B}}$;
+
+\item[(б)] $\boldsymbol{\Gamma} \vDash \boldsymbol{\mathrm{A}} \to \boldsymbol{\mathrm{B}}$\, тады
+і толькі тады, калі\, $\boldsymbol{\Gamma},\, \overline{\boldsymbol{\mathrm{A}}} \vDash \boldsymbol{\mathrm{B}}$;
+
+\item[(в)] калі\, $\boldsymbol{\Gamma},\, \boldsymbol{\mathrm{A}} \vDash \boldsymbol{\mathrm{B}}$\, і\,
+$\boldsymbol{\Gamma},\, \boldsymbol{\mathrm{A}} \vDash \overline{\boldsymbol{\mathrm{B}}}$,\, то\,
+$\boldsymbol{\Gamma} \vDash \overline{\boldsymbol{\mathrm{A}}}$;
+
+\item[(г)] $\boldsymbol{\Gamma} \vDash \boldsymbol{\mathrm{A}} \to (\boldsymbol{\mathrm{B}}
+\to \boldsymbol{\mathrm{C}})$\, тады і толькі тады, калі\, $\boldsymbol{\Gamma} \vDash (\boldsymbol{\mathrm{A}} \cdot \boldsymbol{\mathrm{B}}) \to \boldsymbol{\mathrm{C}}$.
+\end{itemize}
+(Тут $\boldsymbol{\Gamma}$ --- канечнае мноства формул, магчыма пустое.) \\
+\ruText{ (Здесь $\boldsymbol{\Gamma}$ --- конечное множество формул, возможно пустое.) }
+\end{itemize}
+
+
+
+
+\end{document}
+
+
+
+$\boldsymbol{\mathfrak{A}}, \boldsymbol{\mathfrak{B}}, \boldsymbol{\mathfrak{C}}, \boldsymbol{\mathfrak{D}},
```

Comment:
Why this code is placed after \end{document}?

Reply:
I don't know! Removed it - nothing has changed 😂

Reply:
😂😂😂